.github/workflows/node.js.yml

```diff
@@ -8,26 +8,51 @@ on:
     branches: [ "main" ]
 
 jobs:
-  build:
-
+  push_to_registries:
+    name: Push Docker image to multiple registries
     runs-on: ubuntu-latest
-    defaults:
-       run:
-         working-directory: ./
-    strategy:
-      matrix:
-        node-version: [v20.13.1]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
+    permissions:
+      packages: write
+      contents: read
+      attestations: write
     steps:
-    - uses: actions/checkout@v4
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
What is this step for?

Reply:
It was on the documentation so I copied

Reply:
This is for building docker images for multiple platforms. Since we are not building for multiplatform, it can be removed.

```suggestion
```